firmware/common2015/utils/logger/logger.cpp

```diff
@@ -5,18 +5,26 @@
 
 #include <mbed.h>
 #include <rtos.h>
+#include <cstdio>
+#include <errno.h>
 
 const char* LOG_LEVEL_STRING[] = {FOREACH_LEVEL(GENERATE_STRING)};
+const char* LOG_FILE_NAMES[] = {"/local/LOG1.TXT", "/local/LOG2.TXT"};
+
+const int maxLogSize = 500000;  // bytes
```

Comment:
Could you put `500 KB` in this comment (so we don't have to look it up)

Also, with two log files this will take 1MB of space, is that acceptable? I'm not sure how many logs we generate in firmware...